project-base/gitlab/mirror-repository/gitlab-stage.yml

```diff
@@ -0,0 +1,13 @@
+mirror-project-from-github:
+    stage: service
+    only:
+        - web
+        - triggers
+    needs: []
+    script:
+        - apk add git openssh-client
+        - eval $(ssh-agent -s)
+        - mkdir -p ~/.ssh
+        - '[[ -f /.dockerenv ]] && echo -e "Host *\n\tStrictHostKeyChecking no\n\n" > ~/.ssh/config'
+        - echo "$SSHKEY_MIRROR_PROJECT_BASE" | ssh-add -
+        - sh ./gitlab/mirror-repository/mirror.sh "shopsys-project-base" "git@github.com:shopsys/project-base.git" "13.0" "git@gitlab.shopsys.cz:ss6-projects/project-base.git" "devel"
```

Comment:
I have a weird feeling about hardcoding 13.0 here - this should be an ENV variable or something.

Reply:
make sense – changed in 769540a113.
env variable is set in GitLab on CI/CD env variables configuration page